cmd/kube-controller-manager/app/options/attachdetachcontroller.go

```diff
@@ -35,6 +35,7 @@ func (o *AttachDetachControllerOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.DisableAttachDetachReconcilerSync, "disable-attach-detach-reconcile-sync", false, "Disable volume attach detach reconciler sync. Disabling this may cause volumes to be mismatched with pods. Use wisely.")
 	fs.DurationVar(&o.ReconcilerSyncLoopPeriod.Duration, "attach-detach-reconcile-sync-period", o.ReconcilerSyncLoopPeriod.Duration, "The reconciler sync wait time between volume attach detach. This duration must be larger than one second, and increasing this value from the default may allow for volumes to be mismatched with pods.")
+	fs.DurationVar(&o.ReconcilerMaxWaitForUnmountDuration.Duration, "attach-detach-reconcile-max-wait-unmount-duration", o.ReconcilerMaxWaitForUnmountDuration.Duration, "maximum amount of time the attach detach controller will wait for a volume to be safely unmounted from its node. Once this time has expired, the controller will assume the node or kubelet are unresponsive and will detach the volume anyway.")
```

Comment:
How about calling this - `unmount-verification-timeout-on-detach` ?

Reply:
Well, I have no opinion.